### PR TITLE
remove useast hub, aws gateway config changes

### PR DESCRIPTION
**Pull request by scottyhq**
cc @jhamman, @TomAugspurger, @tjcrone. A few changes to dask worker pod config in `pangeo-deploy/values.yaml` matching https://github.com/pangeo-data/pangeo-binder/pull/136. As far as I know this won't impact currently functioning hubs on google and azure, but it does impact all hubs, so I wanted you all to be aware.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Thanks. Seems OK to me.

**Comment by TomAugspurger** [comment]
Failed on Google: https://app.circleci.com/pipelines/github/pangeo-data/pangeo-binder/142/workflows/f1c61f94-7f70-417e-b519-b70e0f1e812d/jobs/146

```
#!/bin/bash -eo pipefail
helm upgrade --wait --install \
  ${CIRCLE_BRANCH} pangeo-binder \
  --namespace=${CIRCLE_BRANCH} --version=v0.2.0 \
  -f ./deploy/${CIRCLE_BRANCH}.yaml \
  -f ./secrets/${CIRCLE_BRANCH}.yaml \
  -f ./secrets/common.yaml

Error: UPGRADE FAILED: timed out waiting for the condition

Exited with code exit status 1

CircleCI received exit code 1
```

Running locally

```
$ helm upgrade --wait --install staging pangeo-binder --namespace=staging --version=v0.2.0 -f ./deploy/staging.yaml -f ./secrets/staging.yaml -f ./secrets/common.yaml
Error: UPGRADE FAILED: rendered manifests contain a new resource that already exists. Unable to continue with update: existing resource conflict: namespace: staging, name: pangeo, existing_kind: /v1, Kind=ServiceAccount, new_kind: /v1, Kind=ServiceAccount
```

Investigating

```
$ kubectl -n staging get serviceaccount                                                                           271ms  Mon Apr  6 09:44:10 2020
NAME                            SECRETS   AGE
binderhub                       1         13d
daskkubernetes                  1         11d
default                         1         390d
hub                             1         13d
pangeo                          1         19m
staging-dask-gateway            1         13d
staging-image-cleaner           1         13d
staging-kube-lego               1         13d
staging-nginx-ingress           1         13d
staging-nginx-ingress-backend   1         13d
```
 
I manually deleted the `pangeo` serviceaccount


```
$ kubectl -n staging delete serviceaccount pangeo 
```

and redeployed

```
$ helm upgrade --wait --install staging pangeo-binder --namespace=staging --version=v0.2.0 -f ./deploy/staging.yaml -f ./secrets/staging.yaml -f ./secrets/common.yaml
```

I deployed again and things seem OK. I wish I new why they failed the first time though.

File: pangeo-deploy/templates/pangeo-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+apiVersion: v1
+metadata:
+  name: pangeo
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: pangeo
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+{{- end -}}
```
Comment (jhamman) [inline pangeo-deploy/templates/pangeo-rbac.yaml:12]: Should this replace the `dask-kubernetes` service account in our helm chart? https://github.com/pangeo-data/helm-chart/blob/master/pangeo/templates/dask-kubernetes-rbac.yaml

Comment (scottyhq) [reply pangeo-deploy/templates/pangeo-rbac.yaml:12]: I think so, but I didn't want to remove it yet in case folks want to keep it around for a bit

Comment (jhamman) [reply pangeo-deploy/templates/pangeo-rbac.yaml:12]: I think we can remove it upstream. People can keep using old tags if they like.

**Pull request merged**